experimental/dds/tree2/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -111,6 +112,7 @@ function rebaseMarkList<TNodeChange>(
 	rebaseChild: NodeChangeRebaser<TNodeChange>,
 	genId: IdAllocator,
 	moveEffects: CrossFieldManager<MoveEffect<TNodeChange>>,
+	nodeExistenceState: NodeExistenceState = NodeExistenceState.Alive,
```

Comment:
If there's only one call site to this function, I don't think we need a default value here.